mfa/methods/code/messaging.md

```diff
@@ -19,18 +19,18 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 
 | Parameter | Required | Description |
 |:--------------|:----------------------|:--|
-| to            | Yes | To telephone number   |
-| from          | Yes | From telephone number |
-| applicationId | Yes | The messaging application id |
+| to            | Yes | The phone number the message should be sent to (must be in E.164 format, like `+19195551212`). |
+| from          | Yes | One of your telephone numbers from which the message should come from (must be in E.164 format, like `+19195551212`). This number must be associated with the messaging application given by the `applicationId` parameter. |
```

Comment:
I lifted this text for the from number from the messaging API. We now know that there's a lot of flexibility in the from on messaging, so we could remove or modify the text around requiring it to be E.164. Perhaps the following:

`One of your telephone numbers from which the message should come from. This number must be associated with the messaging application given by the `applicationId` parameter.`

Reply:
@acovati-bw You had mentioned adding some verbiage around international, something to the effect of 'If sending internationally via IA2P, ....'. Any recommendations here?